Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN yarn run build
 #
 FROM node:16-alpine as production
 USER node
+COPY --chown=node:node --from=base /app/package.json ./package.json
```

Comment:
This is not required in the current implementation right?

Reply:
Changed in [9c26798](https://github.com/5afe/safe-client-gateway-nest/pull/190/commits/9c2679898364e7474e96937607672fd3b810cacb)